Tests/ComparisonTest.cs

```diff
@@ -160,30 +160,49 @@ public static string EvaluatorPrepareEnvironmentCode(Dictionary<string, string>
 
             JSFilenames = null;
 
+            AssemblyAppDomain = AppDomain.CreateDomain("TestAssemblyDomain", null, new AppDomainSetup
```

Comment:
We could probably do what the runner does with js.exe and use a thread to spin up appdomains in advance, assuming that appdomains aren't stop-the-world.

It would probably also be okay to just cache the output of the C# tests, in which case we don't need to run them if they haven't changed. That might be a worthwhile step at this point.